pcsx2/gui/AppCoreThread.cpp

```diff
@@ -357,10 +360,11 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 	wxString gameName;
 	wxString gameCompat;
 	wxString gameMemCardFilter;
+	wxString cheats_ws_archive = Path::Combine(PathDefs::GetProgramDataDir(), wxFileName(L"cheats_ws.zip"));
 
-	int numberLoadedCheats;
-	int numberLoadedWideScreenPatches;
-	int numberDbfCheatsLoaded;
+	int numberLoadedCheats = LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats");
```

Comment:
Updated the code , should be fine now. Assuming that there aren't any more stupid mistakes.